src/devtools/WebAudioEventObserver.test.js

```diff
@@ -27,22 +27,22 @@ describe('WebAudioEventObserver', () => {
     expect(chrome.devtools.network.onNavigated.addListener).toBeCalled();
   });
 
-  it('reattach on unexpected detach', () => {
```

Comment:
@mzgoddard: This test case explicitly checks that the DevTools reattach. Can you explain when unexpected detaches occured or what issue this case was testing against?